assets/js/invoice/invoice.controller.js

```diff
@@ -6,19 +6,18 @@ var InvoiceController = function () {
       detail:displayDetail,
     }
 
-  function displayDetail (id) {
+  $('#invoice-detail').load('/static/js/invoice/templates/invoice_detail.html');
```

Comment:
why is this outside the function? this looks awkward in here (i can't understand the logic behind this line).